from synctogit.service import ServiceError


class EncryptedSectionError(ServiceError):
    pass
